src/app/core/components/left-menu/left-menu.component.html

```diff
@@ -1,4 +1,4 @@
 <alg-left-header></alg-left-header>
 <perfect-scrollbar class="left-nav-theme scrollbar" [ngClass]="{'dark': isNavThemeDark}">
-  <alg-left-nav (themeChange)="onNavThemeChange($event)" (selectId)="onSelectId($event)"></alg-left-nav>
+  <alg-left-nav (useDarkTheme)="onNavThemeChange($event)" (selectId)="onSelectId($event)"></alg-left-nav>
```

Comment:
`useXxx` for an event ? What was wrong with `themeChange` ?

Reply:
I prefer to have a boolean (`useDarkTheme`) than the `string|null` (which was in fact `'dark'|null`) from earlier. 
For the "`useXxx` for an event ", "the component does use dark theme -> yes/no" seems ok to me but I can change it if you prefer. (make proposals)

Reply:
I don't mind the boolean thing, it's more about event naming convention. (Warning, opinionated content below)
For events, as I understand it we use `xx[Change | Update | …]` or `xx[Changed | Updated | …]` convention, but  `useDarkThemeChange`, `darkThemeChange` or `isDarkThemeChange` is not very satisfying from a semantic standpoint.

How about `themeChange(isDark: boolean)` ?

In the end, I think it's not really important, it's just one event :unamused:

Reply:
fixed